File: swapping of two numbers/main1.py
try:
  a = int(input("Enter the value of a : "))
  b = int(input("Enter the value of b : "))
  if a==b:
    print("both are same numbers!")
  else:
    print(f"currently a = {a}\nb = {b}")
    a,b = b,a
    print(f"After swapping: a = {a}, b = {b}")
except ValueError:
  print("Enter valid integer !")

File: C-plus/swapping of two numbers/main 1.cpp
#include <iostream>
using namespace std;
int main(){
    int a,b;
    cout<<"Enter the value of a : ";
    cin>>a;
    cout<<"Enter the value of b : ";
    cin>>b;
    a+=b;b = a-b;a-=b;
    cout<<"after swapping a = "<<a<<", b = "<<b;
}
